app/lib/features/chat_ng/models/chat_editor_state.dart

```diff
@@ -20,7 +20,7 @@ class ChatEditorState {
     MessageAction? actionType,
   }) {
     return ChatEditorState(
-      selectedMsgItem: selectedMsgItem ?? this.selectedMsgItem,
```

Comment:
this assignment is wrong for a reason that after ```unsetActions``` is called, will still fallback to non-null value. Which means the draft would still consider ```messageId``` to be present even when closing preview and mess with the edit/reply states.